modules/covid_research/covid_research.gemspec

```diff
@@ -17,7 +17,7 @@ Gem::Specification.new do |spec|
   spec.files = Dir['{app,config,db,lib}/**/*', 'MIT-LICENSE', 'Rakefile', 'README.md']
   spec.test_files = Dir['spec/**/*']
 
-  spec.add_dependency 'rails', '~> 6.1.5'
+  spec.add_dependency 'rails', '~> 7.0.8'
```

Comment:
Since all modules are running within the master Rails 7.0.8 process, should we just omit the rails version number in modules?

Reply:
Probably?

Reply:
Went ahead and did this ✅